governance/votes/Executive vote - December 10, 2021.md

```diff
@@ -0,0 +1,97 @@
+
+### wstETH-A Parameter Changes
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmYuK441?network=mainnet#poll-detail), the following wstETH-A parameters will be updated if this executive proposal passes.
+
+* Increase the wstETH-A [Maximum Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/module-dciam) from 50 million DAI to 200 million DAI. 
+* Increase the wstETH-A [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam) from 3 million DAI to 20 million DAI.
+* Decrease the wstETH-A [Debt Ceiling Increase Cooldown (`ttl`)](https://makerdao.world/en/learn/governance/module-dciam) from 8 hours to 6 hours.
+* Decrease the wstETH-A [Auction Price Multiplier (`buf`)](https://makerdao.world/en/learn/governance/param-auction-price-multiplier) from 1.3 to 1.2.
+* Increase the wstETH-A [Local Liquidation Limit (`ilk.hole`)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) from 7 million DAI to 15 million DAI.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/wsteth-a-parameters-adjustment-dec-1st-2021/11934) to help inform your position before voting.
+
+### MATIC-A Parameter Changes
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmdzwZyS?network=mainnet#poll-detail), the following MATIC-A parameters will be updated if this executive proposal passes.
+
+* Increase the MATIC-A [Maximum Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/module-dciam) from 20 million DAI to 35 million DAI. 
+* Decrease the MATIC-A [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam) from 20 million DAI to 10 million DAI.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/matic-debt-increase-proposal-for-last-executive-of-2021/11989) to help inform your position before voting.
+
+### Replacing MKR Vesting Streams
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmYdDTsn?network=mainnet#poll-detail), the following MKR Vesting Streams will be canceled and replaced by streams funded by the Protocol Treasury (Pause Proxy) if this executive proposal passes.
+
+1,051.25 MKR will be streamed to the **[Oracles (ORA-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13)** starting July 1, 2021 through July 1, 2022 at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15). A 1-year cliff is in place, making MKR claimable on July 1, 2022.
+
+803.18 MKR will be streamed to the **[Growth (GRO-001) Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP4)** starting July 1, 2021 through July 1, 2022 at **0x7800C137A645c07132886539217ce192b9F0528e** in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP19). A 1-year cliff is in place, making MKR claimable on July 1, 2022.
```

Comment:
MIP40c3-SP15 is their DAI budget proposal. [MIP40c3-SP23](https://mips.makerdao.com/mips/details/MIP40c3SP23) is their MKR budget.

Not sure where that 803.18 is coming from.

Reply:
Great catch on the link, means I got it wrong in the initial exec... better late than never.

On the amount, it is the maximum approved by the voters divided by 4. We decided that would be the simplest way to ensure CUs who do the bulk vesting get access to their funds, while still incentivizing them to be transparent and return unused allocations so they can get their next stream set up.